### ISSUE TITLE
Voltage divider

### ISSUE THREAD

**Issue by RobinMeis** [LilyGO/TTGO-LoRa32-V2.1#1]
Has this version an integrated voltage divider to measure the battery voltage?

**Comment by yeckel** [comment]
Yes on GPIO35 according to the [schematic](https://raw.githubusercontent.com/LilyGO/TTGO-LORA32/master/schematic1in6.pdf).
